src/praisonai/praisonai/suite_runner/runner.py

```
@@ -75,6 +75,26 @@ def _build_env(self) -> dict:
         env.setdefault("PYTHONIOENCODING", "utf-8")
 
         return env
+
+    def _kill_process_tree(self, pid: int) -> None:
+        """Kill a process and its children (needed for uvicorn --reload on Windows)."""
+        if os.name == "nt":
+            subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)